katas/content/oracles/index.md

```diff
@@ -229,7 +229,6 @@ $$\ket{\xi} = \Big[\frac{1}{\sqrt{2}}\big(\ket{110} - \ket{111}\big)\Big] \otime
 
 We can see that these two equations are identical, except for the $-1$ phase that appeared on the $\ket{111}$ basis state - our marked state.  This is a specific example of the phase kickback effect, as the phase from $\ket{-}$ has been *kicked back* into $\ket{x}$.
 
-
```

Comment:
Can we exclude the files with whitespace-only changes from the PR? It does not make sense to do them in files that are otherwise completely unrelated to user-defined types

Reply:
Do they impact any rendering? If not, seems fine to fix while in the files.

If it's just 'noisy' when reviewing, you can suppress by putting `?w=1` on the end of the PR url (e.g. try <https://github.com/microsoft/qsharp/pull/1719/files?w=1>)

Reply:
I can revert those changes if we really want to, but the warnings in those files will then persist and just get fixed again the next time someone saves the file.